src/clang/lib_clang.cr

```diff
@@ -4,8 +4,10 @@ require "./enums"
 {% if flag?(:darwin) %}
   @[Link(ldflags: "-L`xcode-select --print-path`/usr/lib")]
   @[Link(ldflags: "-rpath `xcode-select --print-path`/usr/lib")]
+{% elsif env("LLVM_VERSION") %}
+  @[Link(ldflags: "`llvm-config-{{env("LLVM_VERSION").id}} --ldflags 2>/dev/null`")]
```

Comment:
@olbat I think @ysbaddaden wanted LLVM_CONFIG not LLVM_VERSION as the configurable value.  The caller can then set LLVM_CONFIG location directly both on Mac and linux (and so the conditional should be first).

Reply:
Oops, you're right :-) Done !